src/pages/BlockDetails.vue

```diff
@@ -54,20 +61,21 @@
           </div>
           <div class="blocks-list-page__row">
             <BaseLink :to="`/blocks/${block.height}`" class="block-details-card__row__hashFrame">
-              {{ $t('6 View Change Proofs') }}
+              <!-- TODO wait for design to specify what should happen on click"-->
+              {{ block.view_change_proofs.length + $t(' View Change Proofs') }}
```

Comment:
`vue-i18n` supports some features for interpolation. Read more here:

- [Message Format Syntax](https://vue-i18n.intlify.dev/guide/essentials/syntax.html)
- [Pluralization](https://vue-i18n.intlify.dev/guide/essentials/pluralization.html)

Reply:
Read , Acknowledged and Committed 👍🏼